radiotherapyai/_cli/docs.py

```diff
@@ -16,21 +16,21 @@
 """Building the documentation"""
 
 import os
-import pathlib
 import subprocess
 
-HERE = pathlib.Path(__file__)
-REPO_ROOT = HERE.parents[1]
-DOCS_DIR = REPO_ROOT / "docs"
-SRC_DIR = REPO_ROOT / "src"
-TABLE_OF_CONTENTS_PATH = DOCS_DIR / "_toc.yml"
+from ._paths import DOCS_DIR, REPO_ROOT
 
 
-def build():
+def build(clean: bool):
```

Comment:
should have docstring that explains why one might want to use clean = true (or the converse).  I think the variable name and typical usage makes it clear what is going to happen.

Reply:
Yup, good call. Will do.